core/src/main/scala/slog/LogBuilder.scala

```diff
@@ -0,0 +1,7 @@
+package slog
+
+trait LogBuilder[F[_]] {
```

Comment:
does it ever make sense to have a method that just dumps context without message? for example access logging ... maybe `def nomsg: F[Unit]`

Reply:
underlying logging might not like it.